apps/core/src/hooks/useGetAllBalances.ts

```diff
@@ -14,7 +15,10 @@ export function useGetAllBalances<TResult = CoinBalance[]>(
     const rpc = useRpcClient();
     return useQuery({
         queryKey: ['get-all-balance', address],
-        queryFn: () => rpc.getAllBalances({ owner: address! }),
+        queryFn: async () => {
+            const balances = await rpc.getAllBalances({ owner: address! });
+            return filterOutZeroBalances(balances);
+        },
```

Comment:
nit : use a `select` here 

also just checking that we want to filter out all coins with zero balance across wallet / explorer

Reply:
My understanding is yes, @Jordan-Mysten. This only applies to `getAllBalances`

Reply:
also do we only want to filter out zero balances for unrecognized coins?

Reply:
I think we can, but what informational value are coins with zero balance?